### DISCUSSION TITLE
Addition of Posts, Pages, and Blog Categories

### DISCUSSION THREAD

**Discussion by alsherif-khalaf** [medusajs/medusa#6002] [Feature Requests]
Is there a plugin or a documentation that make the process of addition of  " Posts, Pages, and Blog Categories " easy ?
I would like to propose the addition of Posts, Pages, and Blog Categories as default features.
I believe that integrating these features into MedusaJS would  benefit and contribute to the overall usability and appeal of the framework for a wider audience.

**Comment by alsherif-khalaf** [comment]
any help ?

**Reply by alsherif-khalaf** [reply]
Hello ?

**Reply by olivermrbl** [reply]
Appreciate your suggestion. However, we will not be adding native support for Posts, Pages, and Blog Categories in the near future. I suggest exploring a CMS integration with Medusa, e.g. [Contentful](https://medusajs.com/plugins/medusa-plugin-contentful/), to enable blog functionality on your storefront.  

Alternatively, you can browse community plugins on our [Plugins page](https://medusajs.com/plugins/). If you are lucky, someone has already built a plugin for what you are looking for.

Finally, you have the ability to extend your project with custom entities, API routes, and UI elements, that combined will serve as a blogging feature. The documentation for extending Medusa can be found [here](https://docs.medusajs.com/development/entities/overview).
